docs/getting-started.md

```diff
@@ -116,59 +261,151 @@ That will let you start applications but you still can't interact with them: goo
+```
+IMAGE_INSTALL_append_pn-acrn-image-base = " core-image-base-package"
 ```
-# use the installer wks file
-WKS_FILE_pn-acrn-image-base = "image-installer.wks.in"
 
-# build initramsfs to start the installation
-INITRD_IMAGE_LIVE="core-image-minimal-initramfs"
+To add core-image-weston-package to acrn-image-weston
+```
+IMAGE_INSTALL_append_pn-acrn-image-weston = " core-image-weston-package"
+```
 
-# make sure initramfs and ext4 image are ready before building wic image
-do_image_wic[depends] += "${INITRD_IMAGE_LIVE}:do_image_complete"
-IMAGE_TYPEDEP_wic = "ext4"
+To add core-image-weston-package to acrn-image-base
+```
+IMAGE_INSTALL_append_pn-acrn-image-base = " core-image-weston-package"
+```
 
-# content to be install
-IMAGE_BOOT_FILES_append = "\
-    ${KERNEL_IMAGETYPE} \
-    acrn.bin;esp/acrn.bin \
-    microcode.cpio;esp/microcode.cpio \
-    grub-efi-bootx64.efi;EFI/BOOT/bootx64.efi \
-    ${IMAGE_ROOTFS}/boot/EFI/BOOT/grub.cfg;esp/EFI/BOOT/grub.cfg \
-    ${IMGDEPLOYDIR}/${IMAGE_BASENAME}-${MACHINE}.ext4;rootfs.img \
-"
+### ACRN Configuration
+
+#### ACRN BOARD Configuration
+
+To build for your target board, set `ACRN_BOARD` in your `conf/local.conf`. By default it is set to `nuc7i7dnb`
+```
+ACRN_BOARD = "whl-ipc-i5"
 ```
 
-Now build the installer image:
+Supported Boards:
+- apl-mrb
+- apl-up2-n3350
+- apl-up2
+- nuc6cayh
+- nuc7i7dnb
+- whl-ipc-i5
+- whl-ipc-i7
+
+For More information, Please check [Supported Hardware](https://projectacrn.github.io/latest/reference/hardware.html)
 
+#### ACRN SCENARIO Configuration
+
+To build for your acrn scenario, set `ACRN_SCENARIO` in your `conf/local.conf`. By default it is set to `industry` scenario.
 ```
-$ bitbake mc:installer:acrn-image-base
+ACRN_SCENARIO  = "hybrid"
 ```
+Supported scenarios:
+- sdc
+- logical_partition
+- industry
+- hybrid
+- hybrid_rt
 
+For more information, please check [Build With the ACRN Scenario](https://projectacrn.github.io/latest/getting-started/building-from-source.html#build-with-the-acrn-scenario)
 
-### Override distro configuration
+To customize ACRN Configruation, please check [Introduction to ACRN Configuration](https://projectacrn.github.io/latest/tutorials/acrn_configuration_tool.html)
 
-Due to parsing sequence conflict with meta-intel, weak assingments are not used in acrn-demo-sos and acrn-demo-uos distros. So to override distro configuration in local.conf, override syntex can be used i.e
+#### ACRN BUILD MODE Configuration
 
-WKS_FILE_acrn-demo-sos = "your-custom.wks.in"
+To build ACRN in `RELEASE` mode, set `y` to `ACRN_RELEASE` in your `conf/local.conf`. By default it is set to `n`
+```
+ACRN_RELEASE = "y"
+```
+
+To build ACRN in `DEBUG` mode, set `n` to `ACRN_RELEASE` in your `conf/local.conf`.
+```
+ACRN_RELEASE = "n"
+```
+
+### Kernel Configuration
+
+There are multiple kernel variant available for both SOS and UOS.
+
+#### SOS
+
+To switch to linux-intel-acrn-sos LTS 5.4 kernel (default), in 'local.conf' replace with below lines:
```

Comment:
"replace with below lines" --> "replace with the following lines" (also line 365 and additional occurrences further on)